docs/installation.rst

```diff
@@ -107,9 +107,14 @@ Install from Git repository
 Obtain the Girder source code by cloning the Git repository on
 `GitHub <https://github.com>`_: ::
 
-    git clone https://github.com/girder/girder.git
+    git clone -b 2.x-maintenance https://github.com/girder/girder.git
```

Comment:
This line bothers me, even though it is justified below.  It is unusual to show a `git clone` with the `-b` as part of standard installation instructions.  I think it might be better to have this without the branch, and have the code block showing its use inside the following note.

Reply:
We have a separate page for a [Developer Installation](https://github.com/girder/girder/blob/master/docs/dev-installation.rst) of Girder, where we definitely should encourage people to check out `master`.

However, the line at hand is inside documentation referring to the installation process for general usage, where long-term stability could matter. Under those circumstances, it's far safer to put people on `2.x-maintenance` by default, rather than `master`.

I imagine that those users who are sophisticated enough to appreciate the difference between the two branches will easily be able to modify the `git` command to suit their needs (and honestly would probably be happier if we made the documentation more concise by eliding a lot of these commands). Less sophisticated or less attentive users may also not read the whole note or appreciate its implications, so I'd rather do the "best thing" by default, and place the caveats or alternatives deeper down where intrepid power users can still access them.